src/Gui/DlgSettingsNavigation.ui

```diff
@@ -24,89 +24,122 @@
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
-       <widget class="QLabel" name="stepLabel">
```

Comment:
The grid elements got out of order (because Qt Designer doesn't care about it) -- can you run either @davidosterberg's or my sorter on this? 

- David's (command line): https://github.com/davidosterberg/Qt-UI-file-sorter/
- Mine (GUI): https://github.com/chennes/QtUIXmlCleaner

Reply:
This big diff is due to that I changed the layout type for the navicube pane to use grid layout to allow for aligned columns when I added a new row to it.

I've tried your tool but couldn't install it on macos 10.14.6 (std::filesystem::copy and path seems to require 10.15) and @davidosterberg's tool didn't change the order, it just removed a bunch of `stdset="0"` attributes I've pushed in these changes as an extra clean-up commit before I applied my patches.

Could you try running your (or david's) tool and verify that it rearranges the cell order for you?

Reply:
Apperently `stdset="0"` was important, it fails to build after the tool removes those.

I've removed the prepended cleaning commit to restore the branch to a compilable state.
It would still be interesting to see if you have these issues too.

Reply:
Mine definitely won't work here, I don't support nested grid layouts (sorry, I didn't notice that's what was going on here), but I would think that David's would since he actually parses the XML.

Reply:
> @davidosterberg's tool didn't change the order, it just removed a bunch of `stdset="0"` attributes I've pushed in these changes as an extra clean-up commit before I applied my patches.

I have pushed a new version. I know the PR is closed now. But hopefully the issue is fixed now.

Reply:
Also, removing stdset="0" is slightly preferable when the setter is defined. Because the code would be a tiny bit more efficient. And the xml is shorter. But as you discovered it does not always work. The script allows to keep the stdset by running the command with --no-remove-stdset

Reply:
@davidosterberg nice, I'll try it next time I poke around in UI files. Btw, wwmayer wrote here https://github.com/FreeCAD/FreeCAD/pull/4603#issuecomment-794086886 about why it didn't work without `stdset="0"`.